pages/login.tsx

```diff
@@ -1,36 +1,59 @@
 import Link from "next/link";
 import Head from "next/head";
-import tw, { theme, styled } from "twin.macro";
+import "twin.macro";
 import { useForm } from "react-hook-form";
+import { TwitterLogo, FacebookLogo, LinkedinLogo, SignIn } from "phosphor-react";
+import {
+    Button,
+    ButtonProps,
+    Container,
+    Grid,
+    Text,
+    Heading,
+    Divider,
+    IconButton,
+    VStack,
+} from "@chakra-ui/react";
 
-import { ImFacebook, ImGoogle, ImLinkedin } from "react-icons/im";
-import { FaSignInAlt } from "react-icons/fa";
+import { useScreen } from "hooks";
+import { InputField } from "components/units/InputField";
 
-import { Button, InputField } from "components/units";
-import { useMinScreen } from "hooks";
+const data = [
+    { name: "Facebook", Icon: FacebookLogo, color: "facebook" },
+    { name: "Twitter", Icon: TwitterLogo, color: "twitter" },
+    { name: "LinkedIn", Icon: LinkedinLogo, color: "linkedin" },
+] as const;
 
 const SocialSignIn = () => {
-    const { min } = useMinScreen();
+    const { min, max } = useScreen();
+
+    const buttonProps: ButtonProps = {
+        w: "full",
+        size: "lg",
+        fontSize: "sm",
+    } as const;
```

Comment:
Same as above comment, I'm sure there's a reason you have to do this but I'm curious as to why?

Reply:
See my reply above